### DISCUSSION TITLE
Auto-fill option?

### DISCUSSION THREAD

**Discussion by tomhusband** [Eloston/ungoogled-chromium#2550] [Q&A]
I'm trying out ungoogled-chromium on my Linux Mint laptop. I've installed the Web Store extenstion and have started installing my extensions. First up is Bitwarden. It installed fine but when I try autofilling on a site it's not available. I'm used to ctrl+shift+l to fill in my password. Having to copy and paste is a pain. I must be missing a setting somewhere.

Any hints appreciated. Many thanks.

**Comment by PF4Public** [comment]
Maybe this shortcut needs to be configured first?

**Comment by tomhusband** [comment]
I guess this can marked complete as I just checked again and keyboard shortcuts seem to working. The shortcut is the Bitwarden default so I must have ticked something.

Anyway, all is well.
